h2o-py/h2o/grid/metrics.py

```diff
@@ -41,6 +56,19 @@ def F1(self, thresholds=None, train=False, valid=False, xval=False):
         :param bool valid: If True, return the F1 value for the validation data.
         :param bool xval: If True, return the F1 value for each of the cross-validated splits.
         :returns: Dictionary of model keys to F1 values
+
+        :examples:
+
+        >>> from h2o.grid.grid_search import H2OGridSearch
+        >>> from h2o.estimators.glm import H2OGeneralizedLinearEstimator
+        >>> hyper_parameters = {'alpha': [0.01,0.5], 'lambda': [1e-5,1e-6]}
+        >>> gs = H2OGridSearch(H2OGeneralizedLinearEstimator(family='binomial'),
+        ...                                                  hyper_parameters)
+        >>> training_data = h2o.import_file("http://s3.amazonaws.com/h2o-public-test-data/smalldata/logreg/benign.csv")
```

Comment:
for these examples, can you move the importing of data to after the import statements?